docs/user_manual/qcs.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: "20. QCs"
+section: 20
+---
+
+QCs in MISO are used to record the results of quality control checks. They can be added to samples, libraries, pools,
+and sequencing containers. It is possible to add the same QC to the same item multiple times to show a change in the
```

Comment:
Unclear to me how adding the /same/ QC to an item multiple times would show a change. Knowing little about the feature, 1 QC to me seems like 1 record of a quality control check having been completed, so individual QCs should not change (that'd be rewriting history). Could the data representation here be elaborated on / is it explained elsewhere?

Reply:
Reworded to clarify that multiple QCs of the same type can be added, rather than multiple of the same QCs